torch/quantization/fx/quantize.py

```diff
@@ -955,6 +967,14 @@ def record_match(pattern, node, matched):
         return match_map
 
     def _find_quants(self, graph, matches):
+        """
+        Inputs:
```

Comment:
would also be good to describe what this is doing.
this function finds the input and output nodes of quantizable module/functional/torch ops that also needs to be quantized and return a map from node name to a tuple of QuantizeHandler and qconfig